benchmark/fs/bench-stat-promise.js

```diff
@@ -14,7 +14,7 @@ async function run(n, statType) {
   let remaining = n;
   bench.start();
   while (remaining-- > 0)
-    await fsPromises[statType](arg);
+    await (statType === 'fstat' ? arg.stat() : fsPromises[statType](arg));
```

Comment:
It would be good to move the condition up to only do `statType === 'fstat'` once.